proposals/argocd-vault-plugin.md

```diff
@@ -0,0 +1,15 @@
+Welcome to Argo Project Onboarding!
+
+Before submitting the ticket please ensure you understand which [projects](../../community/ecosystem-projects.md) could be added to the Argo community and what the open decision-making process looks like.
+
+Once you are ready, please help the reviewer understand your project better by
+answering the following questions in your onboarding proposal:
+
+- [x] What is your project repository Github URL? - **https://github.com/IBM/argocd-vault-plugin**
+- [x] Do you wish to host your project repository https://github.com/argoproj-labs or https://github.com/argoproj? - **https://github.com/argoproj-labs**
+- [x] Does your project focus on enhancing or providing additional features to one of the existing [core
+ projects](https://github.com/argoproj/argoproj#what-is-argoproj) ? If yes, which of the core projects is your proposed project related to? - **https://github.com/argoproj/argo-cd**
+- [x] Is it endorsed by any of the Argo subproject maintainers? Please mention sponsors from the subproject. - **@sbose78**
+- [x] How does it align with the goals of the Argo community? - **argocd-vault-plugin adds an unofficial secret management plugin that can be used in tandem with Argo CD or could be used independently as part of an Argo Workflow.** 
```

Comment:
Could you please share a detailed example of how this could be used with Argo CD ?